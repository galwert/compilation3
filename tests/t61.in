void main()
{
    int x = 100;
    byte y = 100b;

    if (x>y)
    {
        return;
    }
    bool xSucks = x < y;
    if (xSucks)
    {
        return;
    }
    else
    {
        return;
    }

    bool tmp1 = x == y;
    bool tmp2 = x != y;
    bool tmp3 = (int)y == (byte)x;
    bool tmp4 = true;
    if (true == false) // mismatch error (for some unknown god damn reason) [PDF page 4 rule 9]
    {
        print("....How..?");
    }
}