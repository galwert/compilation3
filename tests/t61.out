---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
line 24: type mismatch
